styles.js

```diff
@@ -10,5 +10,17 @@ export default createGlobalStyle`
   body {
     margin: 0;
     font-family: system-ui;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: #cce0d2;
   }
+
+  h1, h2 {
+    text-align: center;
+  }
```

Comment:
the h2 text align center will cause problems in the future when you forgot that you put it here. Please move it to a styled component instead.